src/coverlet.core/Reporters/IReporter.cs

```diff
@@ -1,14 +1,14 @@
 namespace Coverlet.Core.Reporters
 {
-    public interface IReporter
+    internal interface IReporter
```

Comment:
I'm not sure in what situations coverlet would be used as a lib, but could one be that the user want to provide their own IReporter?  Or would they just get CoverageResult directly from coverlet and do whatever they need to do with that without going through the reporting interface?

Reply:
In past we had a discussion on it https://github.com/tonerdo/coverlet/issues/212 some guys wants integrate the feature as native.

Reply:
Do you agree?What do you think?

Reply:
Ok, so for now hide everything, then in a separate issue start designing externally viable APIs?  Sounds good.

Reply:
Yep